src/Tooltip/TooltipContent.js

```diff
@@ -51,9 +51,11 @@ class TooltipContent extends PureComponent {
     if (position === 'top' || position === 'bottom') {
       top +=
         (position === 'top' ? -tooltipHeight : triggerHeight) + sign * offsetY
-      left += calculateAlignment(triggerWidth, tooltipWidth, align)
+      left +=
+        calculateAlignment(triggerWidth, tooltipWidth, align) + sign * offsetX
```

Comment:
1. `sign` is used to calculate space between tooltip and trigger element for opposite cases, e.g. `top/bottom`, `left/right`. For tooltip positioned at the top it will be `-10px` from the top, for tooltip positioned at the bottom it's `10px` from the bottom. Alignment calculation is invariant to positioning system.

2. If we want to expose such functionality, props should have a reasonable names: `positionOffset` and `alignOffset`